adapters/syncer_test.go

```diff
@@ -0,0 +1,35 @@
+package adapters
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func TestSyncers(t *testing.T) {
+	syncers := make(map[openrtb_ext.BidderName]usersync.Usersyncer)
+	for _, bidderName := range map[string]openrtb_ext.BidderName{} {
```

Comment:
Not forgotten about, see my comment above. The map is ultimately moved to the adapters package, however importing the static map from where it lives now would cause a circular dependency.

Reply:
aha, yeah that makes sense. I wasn't sure which one of the two tests you wanted to be the "real" one.

Can you just delete it? It doesn't make much sense to have tests without assertions.